### PR TITLE
Assets: CSS compiling

### PR DESCRIPTION
**Pull request by PhillippOhlandt**
Ok definitely a topic we can discuss about but I think the solution with laravel-elixir is the easiest one for any user. And at the end of the day, this is just a recommendation by me and can be changed by any time from the user.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
+/node_modules
 .DS_*
 config.yml
 composer.lock
 vendor/
 tests/tmp/
+/assets/extension.css.map
```
Comment (GwendolenLynch) [inline .gitignore:8]: Probably want `bower_components` too, and you can drop the leading `/`

**Comment by GwendolenLynch** [comment]
So… how to put this… :stuck_out_tongue_winking_eye: 

The prevailing opinion of the core team goes something like "Cramming NodeJS into a PHP framework is just, wrong"… 

Elixir itself is another one one of Taylor's "solutions" looking for a problem to solve, and none of this belongs in an extension.

**Comment by PhillippOhlandt** [comment]
First, not Taylor coded that, that was Jeffrey :P
Second, I think no one will write CSS by hand anymore xD
Third, for that, NodeJS is standard to compile the stuff

And so on and so on. I mean you can do it all by hand but then you require 70 node packages and your gulpfile is very big and confusing. 

I don't get the "NodeJS into a PHP framework is just, wrong" part. Because it is just for the assets. And when you say this, you have to remove all the stuff from Bolt itself, or? :P

**Comment by GwendolenLynch** [comment]
No, what is being said here is that this doesn't belong in an extension itself.

**Comment by GwendolenLynch** [comment]
Oh and yeah, your other points are valid… using tools to compile this stuff is the way to go… 

It is just not what we believe should be the approach of this extension.

**Comment by PhillippOhlandt** [comment]
Ok, then no assets management for users.

**Pull request closed without merging**